### PR TITLE
Plugin cleanup

### PR DESCRIPTION
**Pull request by Matt07211**
This is general cleanup, that is removing unneeded files (aka files that have been fully commented out). ~~as well as removing the German specific language option (would rather let translations handle anything like this). Also choose to drop Anisearch and Proxer in favor of maintaining just the big 3 (AniDB, Anilist and MAL). And in all honest, I haven't heard of those two outside of the plugin, if you have a good and appropriate reason to add them back in please voice your concerns.~~  Concerns have been voiced, the metadata agent removal has been reverted

With this out of the way, time to fix up the plugins bugs and add some more features to it.

### CODE REVIEW

File: MediaBrowser.Plugins.Anime/Configuration/PluginConfiguration.cs [hunk 1]
```diff
@@ -35,8 +30,8 @@ public class PluginConfiguration
         public bool AddAnimeGenre { get; set; }
         public bool UseAnidbOrderingWithSeasons { get; set; }
         public string MyAnimeList_API_Name { get; set; }
-        public string MyAnimeList_API_Pw { get; set; }
-        public int AniDB_wait_time { get; set; }
+	public string MyAnimeList_API_Pw { get; set; }
+	public int AniDB_wait_time { get; set; }
```
Comment (Bond-009) [inline MediaBrowser.Plugins.Anime/Configuration/PluginConfiguration.cs]: Weird indentation

File: MediaBrowser.Plugins.Anime/anime-lists-net/AnimelistAnime.cs [hunk 2]
```diff
@@ -26,14 +26,6 @@ public class AnimelistAnime
         [XmlAttribute("anidbid")]
         public string AnidbId { get; set; }
 
-        /// <remarks />
-        [XmlAttribute("anisearch")]
-        public string AniSearchId { get; set; }
```
Comment (Bond-009) [inline MediaBrowser.Plugins.Anime/anime-lists-net/AnimelistAnime.cs]: Why remove these?

File: MediaBrowser.Plugins.Anime/Providers/Proxer/ProxerSeriesProvider.cs [hunk 3]
```diff
@@ -1,163 +0,0 @@
...
-using Microsoft.Extensions.Logging;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace MediaBrowser.Plugins.Anime.Providers.Proxer
-{
-    public class ProxerSeriesProvider : IRemoteMetadataProvider<Series, SeriesInfo>, IHasOrder
```
Comment (Bond-009) [inline MediaBrowser.Plugins.Anime/Providers/Proxer/ProxerSeriesProvider.cs]: Why delete this file?

**Review by Bond-009 — CHANGES_REQUESTED** [review]
Don't know why, but you are deleting files that weren't commented out.

**Comment by Matt07211** [comment]
Re: Discussed files above, I was in favour of just maintaining the big 3 metadata and list provider's, but others would rather keep them in at this time, I'll revert those changes and add them back in tonight

File: MediaBrowser.Plugins.Anime/Configuration/PluginConfiguration.cs (hunk 1 above)
Comment (Matt07211) [reply MediaBrowser.Plugins.Anime/Configuration/PluginConfiguration.cs]: Fixed

File: MediaBrowser.Plugins.Anime/Providers/Proxer/ProxerSeriesProvider.cs (hunk 3 above)
Comment (Matt07211) [reply MediaBrowser.Plugins.Anime/Providers/Proxer/ProxerSeriesProvider.cs]: Reverted change.

File: MediaBrowser.Plugins.Anime/anime-lists-net/AnimelistAnime.cs (hunk 2 above)
Comment (Matt07211) [reply MediaBrowser.Plugins.Anime/anime-lists-net/AnimelistAnime.cs]: Reverted change.

**Comment by Matt07211** [comment]
As per discussion in the matrix I have reverted the changes that removed 2 metadata providers, so all this pull request should now be doing is removing completely useless and unused code.

**Comment by hawken93** [comment]
LGTM :)

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**